config/software/gecode.rb

```diff
@@ -25,6 +25,11 @@
   source md5: "b4191d8cfafa18bd9b78594544be2a04"
 end
 
+# Major version, have not tried yet
+version "4.4.0" do
```

Comment:
It's fine to make this available as we aren't changing the default_version, but I don't see us moving to this anytime soon/ever.